e2e/cypress/support/commands.ts

```diff
@@ -36,7 +36,7 @@ Cypress.Commands.add('applyProjectsFilter', (projectsToFilterOn: string[]) => {
     .find('chef-checkbox').each(child => {
     // deselect every checkbox
     if (child.attr('aria-checked') === 'true') {
-      child.click();
+      child.trigger('click');
```

Comment:
got a lint warning about this: the `child` is an HTML element, so we use JQuery instead of Cypress to click on it. JQuery has deprecated `click()` in favor of `trigger("click")`